docs/pages/kubernetes-access/discovery/aws.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Teleport EKS Auto-Discovery (Preview)
+description: Auto-Discovery of EKS clusters in AWS cloud.
+---
+
+Similarly to [AKS Auto-Discovery](./azure.mdx), EKS Auto-Discovery can automatically 
+discover any EKS cluster and enroll it in Teleport if its tags match the 
+configured labels. 
+
+(!docs/pages/kubernetes-access/discovery/includes/step-description.mdx!)
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An AWS account with permissions to create and attach IAM policies.
+- An AWS account with `system:masters` RBAC access to EKS clusters.
+- A host to run the Teleport Discovery and Kubernetes services.
+- One or more EKS clusters running. 
+
+## Known limitations
```

Comment:
Think it is worth it to link to that aws-auth API ticket from EKS roadmap here? Maybe in an admonition/note or something. And say that once EKS team implements it, we'll be able to improve this in future. Maybe it will drive more upvotes on that ticket as well :)

Reply:
Added it to the bottom of Known limitations section